app.py

```diff
@@ -19,26 +18,49 @@ def index():
     return {"message": data}
 
 
-
 @app.route('/recommend/popular')
```

Comment:
The function 'unquote' is called without importing it. Ensure that the 'urllib.parse' module is imported.